packages/builder/src/pages/builder/auth/login.svelte

```diff
@@ -11,7 +11,7 @@
   } from "@budibase/bbui"
   import { goto } from "@roxi/routify"
   import { auth, organisation, oidc, admin } from "stores/portal"
-  import GoogleButton from "./_components/GoogleButton.svelte"
+  import GoogleButton from "components/common/GoogleButton.svelte"
```

Comment:
This button needs to be switched back to the one at the previous location `./_components/GoogleButton.svelte`. The updated button is used in Google authorisation flows within the builder and cannot function properly/display in the login page. 

![Screenshot 2023-06-12 at 14 32 40](https://github.com/Budibase/budibase/assets/5913006/ce7c2f55-a430-4f66-89d1-1b11ba43a170)

Reply:
Jeepers, my bad. Dodgy find and replace I guess 🤦